apps/fabric-website/webpack.serve.config.js

```diff
@@ -1,5 +1,6 @@
 let path = require('path');
```

Comment:
>let [](start = 0, length = 3)

Just curious, should this file name be "webpack.server.config.js"? Or "webpack.serve.config.js" is correct?

Reply:
"webpack.serve.config.js" is correct here

---
In reply to: [302627456](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302627456) [](ancestors = 302627456)

Reply:
"serve" is standard in our projects as well as various other open source projects.